.github/workflows/sonar_pr_analysis.yml

```diff
@@ -0,0 +1,80 @@
+---
+name: Sonar analysis for pull requests
+
+on:
+  workflow_run:
+    workflows:
+      - Pull request verification
+    types:
+      - completed
+env:
+  JAVA_DISTRIBUTION: 'temurin'
+  JAVA_VERSION: '17'
+
+# Cancel in-progress sonar branch analysis workflows. We only care about analyzing the latest commit.
+concurrency:
+  group: ${{ github.event.workflow.name }}-${{ github.ref_name }}
```

Comment:
Is the github.ref_name variable available here? Since the workflow wasn't triggered by a push on a branch/tag or PR commit, how can it have a branch name in the context?

Reply:
Perhaps it should use the `github.event.workflow_run.pull_requests[0].number` variable like below?

Reply:
Done. Switched to number.